test/spec/modules/mobilefuseBidAdapter_spec.js

```diff
@@ -0,0 +1,107 @@
+import { expect } from 'chai';
+import { spec } from 'modules/mobilefuseBidAdapter.js';
+
+const bidRequest = {
+  bidder: 'mobilefuse',
+  params: {
+    placement_id: 'test-placement-id',
+    bidfloor: 1.25,
+  },
+  adUnitCode: 'ad-slot-1',
+  mediaTypes: {
+    banner: {
+      sizes: [[300, 250]],
+    }
+  },
+  bidId: 'bid-id-123',
+  transactionId: 'txn-id-123',
+  gpid: 'test-gpid',
+  userIdAsEids: [{
+    source: 'sharedid.org',
+    uids: [{ id: '01ERJ8WABCXYZ6789', atype: 1 }],
+  }],
+  schain: {
+    ver: '1.0',
+    complete: 1,
+    nodes: [{ asi: 'exchange.com', sid: 'abc123', hp: 1 }],
+  },
+};
+
+const bidderRequest = {
+  bidderCode: 'mobilefuse',
+  bids: [bidRequest],
+  uspConsent: '1YNN',
+  gppConsent: {
+    gppString: 'GPP_CONSENT_STRING',
+    applicableSections: [7],
+  },
+};
+
+const serverResponse = {
+  body: {
+    seatbid: [{
```

Comment:
adomain is required, please write a test demonstrating it makes it to bid.meta.advertiserDomains

Reply:
Added a test. Let me know if that's sufficient.